File: Selenium/Activities/Activity2.py
from selenium import webdriver
from selenium.webdriver.common.by import By

driver = webdriver.Edge()

driver.get("https://training-support.net/webelements/login-form/")
print(driver.title)

driver.find_element(By.ID, "username").send_keys("admin")
driver.find_element(By.ID, "password").send_keys("password")

driver.find_element(By.XPATH, "//button[text()='Log in']").click()
driver.delay(30)
driver.quit()

File: Selenium/Project/Activity2.java
package examples;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity2 {
	public static void main(String[] args) {
		WebDriver driver = new FirefoxDriver();
		
		driver.get("https://training-support.net/webelements/login-form");
		
		System.out.println("Title of page is: " + driver.getTitle());
		
		WebElement username= driver.findElement(By.id("username"));
		username.sendKeys("admin");
		
		WebElement password= driver.findElement(By.name("password"));
		password.sendKeys("password");
		
		WebElement button = driver.findElement(By.xpath("//button[text()='Submit']"));
		button.click();

		driver.quit();
	}
}